core/files/configure_misp.sh

```diff
@@ -6,10 +6,20 @@ source /rest_client.sh
 [ -z "$GPG_PASSPHRASE" ] && GPG_PASSPHRASE="passphrase"
 [ -z "$REDIS_FQDN" ] && REDIS_FQDN="redis"
 [ -z "$MISP_MODULES_FQDN" ] && MISP_MODULES_FQDN="http://misp-modules"
+[ -z "$OIDC_PROVIDER_URL" ] && OIDC_PROVIDER_URL="test_provider"
```

Comment:
Not sure whether we want default values if not working

Reply:
Let me rephrase it. I would rather remove default values, and add some checks when setting up oidc (e.g., if not all values are set, error out).

Reply:
Done and added a generic  utility which can be used for other configuration functions in the future

Reply:
Nice. I see the output is redirected to stderr. Should we leave it like that?